backend/Pipfile

```diff
@@ -31,6 +31,7 @@ django-extensions = "==2.1.6"
 django-cors-headers = "==3.1.1"
 isort = "==4.3.17"
 black = "==18.9b0"
+httpretty= "*"
```

Comment:
Is it possible to set a fixed version for `httpretty`? You can use the latest (looks like it’s [0.9.7 according to Pypi](https://pypi.org/project/httpretty/0.9.7/))